static/third/spectrum/spectrum.js

```diff
@@ -28,7 +28,7 @@
         cancelText: "cancel",
         chooseText: "choose",
         preferredFormat: false,
-        className: "",
+        className: "new-style",
```

Comment:
We presumably don't need this change -- we can just pass it in as an option, right?

Reply:
Hmm, let me check.

Reply:
I think this is the right place to pass the class name. The other places I could think of is -

``` 
container.addClass(opts.className);
```